packages/manager/src/features/Kubernetes/CreateCluster/CreateCluster.tsx

```diff
@@ -57,67 +56,84 @@ import { NodePoolPanel } from './NodePoolPanel';
 
 import type {
   CreateKubeClusterPayload,
-  CreateNodePoolData,
   KubeNodePoolResponse,
   KubernetesTier,
 } from '@linode/api-v4/lib/kubernetes';
-import type { Region } from '@linode/api-v4/lib/regions';
 import type { APIError } from '@linode/api-v4/lib/types';
-import type { ExtendedIP } from 'src/utilities/ipUtils';
 
 export const CreateCluster = () => {
   const { classes } = useStyles();
-  const [selectedRegion, setSelectedRegion] = React.useState<
-    Region | undefined
-  >();
-  const [nodePools, setNodePools] = React.useState<KubeNodePoolResponse[]>([]);
-  const [label, setLabel] = React.useState<string | undefined>();
   const [version, setVersion] = React.useState<string | undefined>();
   const [errors, setErrors] = React.useState<APIError[] | undefined>();
-  const [submitting, setSubmitting] = React.useState<boolean>(false);
   const [hasAgreed, setAgreed] = React.useState<boolean>(false);
   const formContainerRef = React.useRef<HTMLDivElement>(null);
   const { mutateAsync: updateAccountAgreements } = useMutateAccountAgreements();
-  const [highAvailability, setHighAvailability] = React.useState<boolean>();
   const [controlPlaneACL, setControlPlaneACL] = React.useState<boolean>(false);
-  const [apl_enabled, setApl_enabled] = React.useState<boolean>(false);
-
+  const [submitting, setSubmitting] = React.useState<boolean>(false);
   const { data, error: regionsError } = useRegionsQuery();
   const regionsData = data ?? [];
   const history = useHistory();
   const { data: account } = useAccount();
   const { showAPL } = useAPLAvailability();
   const { showHighAvailability } = getKubeHighAvailability(account);
   const { showControlPlaneACL } = getKubeControlPlaneACL(account);
-  const [ipV4Addr, setIPv4Addr] = React.useState<ExtendedIP[]>([
-    stringToExtendedIP(''),
-  ]);
-  const [ipV6Addr, setIPv6Addr] = React.useState<ExtendedIP[]>([
-    stringToExtendedIP(''),
-  ]);
   const [selectedTier, setSelectedTier] = React.useState<KubernetesTier>(
     'standard'
   );
 
+  const {
+    isLoadingVersions,
+    versions: versionData,
+    versionsError,
+  } = useLkeStandardOrEnterpriseVersions(selectedTier);
+
+  const versions = (versionData ?? []).map((thisVersion) => ({
+    label: thisVersion.id,
+    value: thisVersion.id,
+  }));
+
+  const formValues = React.useMemo(() => {
+    return {
+      apl_enabled: false,
+      hasAgreed,
+      tier: 'standard' as KubernetesTier,
+    };
+  }, [hasAgreed]);
+
+  const { control, getValues, handleSubmit, setValue, watch } = useForm<
+    Required<CreateKubeClusterPayload>
+  >({
+    defaultValues: formValues,
+  });
```

Comment:
I feel like `Required` shouldn't be necessary here.

```suggestion
  const {
    control,
    getValues,
    handleSubmit,
    setValue,
    watch,
  } = useForm<CreateKubeClusterPayload>({
    defaultValues: formValues,
  });
```